### PR TITLE
Populate landing page links

### LINKED ISSUE #357: Populate Front Page Links
None of the links work on the staging front page.

### PR DESCRIPTION
**Pull request by nimbinatus**
Had to use an anchor instead of the button for the link to work in FF :slightly_frowning_face:

fixes #357

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ customers happy.
     <span class="card-heading">For Developers</span>
     <p>Use these resources to learn and implement Helix patterns.</p>
     <ul>
-      <li><a href="#">Pattern Guidelines</a></li>
-      <li><a href="#">UI Framework</a></li>
-      <li><a href="#">Component Explorer</a></li>
+      <li><a href="http://helix.rax.io/">Pattern Guidelines</a></li>
```
Comment (technabors) [inline index.md]: I wonder if we remove this link for the time being. Seems like all the links in this card should go to the helix-ui side, but there isn't really anything that related to pattern guidelines at the moment. 

Thoughts @araiford

File: index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ customers happy.
     </ul>
     <div class="card-bottom">
       <div class="card-button">
-        <button class="hxBtn hxBtn--primary"><a href="{{site.baseurl}}getting-started/design.html">Full Designer Guide</a></button>
+        <a class="hxBtn hxBtn--primary" href="{{site.baseurl}}getting-started/design.html">View Designer Guide</a>
```
Comment (nimbinatus) [inline index.md]: From code review: remove card-bottom buttons altogether; card min-height to 20rem

**Comment by nimbinatus** [comment]
Here's what the new version looks like:
screen shot 2017-11-20 at 10 42 34 am

**Comment by nimbinatus** [comment]
@araiford @technabors @palindromebart ^^ ready for review

File: index.md [hunk 3]
```diff
@@ -12,30 +12,25 @@ customers happy.
...
-    </ul>
+    <p>Use these resources to implement Helix code.</p>
     <div class="card-bottom">
-      <div class="card-button">
-        <button class="hxBtn hxBtn--primary"><a href="https://rackerlabs.github.io/helix-ui/guides/install/">Full Developer Guide</a></button>
-      </div>
+      <ul>
+        <li><a href="https://github.com/rackerlabs/helix-ui/">Code Repository</a></li>
+        <li><a href="https://rackerlabs.github.io/helix-ui/guides/install/" target="_blank">Installation Guide</a></li>
+        <li><a href="https://rackerlabs.github.io/helix-ui/" target="_blank">Component Explorer</a></li>
```
Comment (araiford) [inline index.md]: Add the external link indicator to the `<a>` tag these that have target="_blank"

`Installation Guide <hx-icon type="external-link"></hx-icon>`

**Pull request merged**